app/views/support_interface/candidates/edit_candidate_account_status.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :before_content, govuk_back_link_to(support_interface_candidate_path, 'Back') %>
```

Comment:
How does this path magically know the ID of the candidate? 🤯

Reply:
It uses the referer if the url doesn't exist. I will amend the candidate to the url (or just put `:back` - I'll see).

Reply:
Done!